test/e2e/e2e.go

```diff
@@ -176,7 +64,7 @@ var _ = ginkgo.SynchronizedAfterSuite(func() {
 		}
 	}
 	if framework.TestContext.NodeKiller.Enabled {
-		close(nodeKillerStopCh)
```

Comment:
@andrewsykim I added it back, but I don't know if I can really access the `NodeKillerStopCh` channel on e2e.go.